tests/js/server/resilience/moving-shards-with-arangosearch-view-cluster.js

```diff
@@ -58,25 +58,59 @@ function MovingShardsWithViewSuite (options) {
   var dbservers = getDBServers();
   var useData = options.useData;
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief returns "{shardXY: [serverAB, ...], ...}" object of the given
+/// collection from the plan
+////////////////////////////////////////////////////////////////////////////////
+
+  function collectionShardInfo(database, collection) {
+    if (typeof collection !== "string") {
+      throw new Error(`argument is not a string: ${collection}`);
+    }
+    var cinfo = global.ArangoClusterInfo.getCollectionInfo(database, collection);
+    return cinfo.shards;
+  }
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief returns "{shardXY: serverAB, ...}", with the leader of each shard
+/// from the plan
+////////////////////////////////////////////////////////////////////////////////
+
+  function shardLeaders(database, collection) {
+    const shardInfo = collectionShardInfo(database, collection);
+
+    return Object.keys(shardInfo).reduce(function(result, shard) {
+      const servers = shardInfo[shard];
+      assertTrue(servers.length > 0);
+      result[shard] = servers[0];
```

Comment:
The first DBServer is leader in `global.ArangoClusterInfo.getCollectionInfo(database, collection)`?

Reply:
Yes, the leader is the first server in the Plan.